src/slack_api_util.ts

```diff
@@ -26,13 +26,48 @@ type SlackSendMessageResponse = {
 type SlackSendMessageOptions = {
   channel: string;
   parentMessageTs?: string;
+  parse?: boolean;
+  unfurl_links?: boolean;
   blocks?: SlackBlock[];
 };
 
 type SlackChannelNamesAndIds = {
   [channelId: string]: string; // mapping of channel ID to channel name
 };
 
+export async function getThreadPermalink(
```

Comment:
Can we have the SlackApiUtil function just do the Slack API call and have the DB lookup separate?